.github/workflows/ci.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       matrix:
         node-version: [12.x]
-        rust: ['1.70.0']
+        rust: ['1.74.1']
```

Comment:
We can remove the `rust: ['1.74.1']` and the following "Install Rust" step from workflow files.

GitHub Action runners default to have Rust. And `rust-toolchain.toml` will automatically take effect.

Therefore, we will only need the Rust version in `rust-toolchain.toml` which is more maintainable.